content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Introduction to Integrations
+kind: documentation
+further_reading:
+- link: "https://learn.datadoghq.com/"
+  tag: "Learning Center"
+  text: "Introduction to Datadog"
+- link: "/integrations/"
+  tag: "Integrations"
+  text: "Datadog's full list of integrations"
+---
+
+This is a guide for using integrations, for information about building a new integration, see the [Create a new integration][1] page.
+
+An integration, at the highest level, is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole—you can see pieces individually and also how individual pieces are impacting the whole.
+
+* Over 250 Agent integrations are supported in the [integrations core][2].
+* Even more Agent integrations have been built by our open source community, in [integrations extras][3].
+* Find all available integrations on the [Integrations page in the app][4].
+* You can build your own integration using the [Datadog API][5].
+* The Agent is [open source][6] and you can [create your own custom check][7].
+
+Datadog provides three main types of integrations:
+
+* Agent-based integrations are installed with the Datadog Agent and use a Python class called `check` to define the metrics to collect.
+* Authentication (crawler) based integrations are set up in the [Datadog App][8] where you provide credentials for obtaining metrics with the API. These include popular integrations like [Slack][9],[AWS][10],[Azure][11], and [PagerDuty][12].
+* Library integrations allow you to monitor applications based on the language they are written in, like [Node.js][13], or [Python][14].
+
+It's best to start collecting metrics on your projects as early in the development process as possible, but you can start at any stage.
+
+### Key terms
+
+Find below the key terms for Datadog integrations:
+
+| Term         | Description |
+| ------------ | ----------- |
+|**parameters** | Use the parameters in the `conf.yaml` file to control accesses between your integration data source and the Agent. The individual integrations `conf.yaml.example` file has all of the required and not required parameters listed. Define all of the needed parameters for all of your integrations in a single `conf.yaml` file - if you have more than one file, the Agent only uses the alphabetically first file. |
+|**conf.yaml** | You create the `conf.yaml` in the `conf.d/` folder at the root of your [Agent's configuration directory][15]. Use this file to connect integrations to your system, as well as configure their settings. Nonrequired parameters are usually commented out.|
```

Comment:
I'd mention the `integration_name.d` folder as well, like https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6#checks-configuration-files-for-agent-6